lib/games.js

```diff
@@ -2363,24 +2363,12 @@ export const games = {
     }
   },
   redorchestra: {
-    name: 'Red Orchestra',
-    release_year: 2018,
-    options: {
-      port: 7758,
-      port_query_offset: 1,
-      protocol: 'unreal2'
-    }
-  },
```

Comment:
We can't remove this entry, even if it is completely wrong and if it is, we should remove it in a major version.